src/Nethermind/Nethermind.Serialization.Rlp/MyTxDecoder/ITxDecoder.cs

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using Nethermind.Core;
+
+namespace Nethermind.Serialization.Rlp.MyTxDecoder;
+
+interface ITxDecoder
+{
+    public const int MaxDelayedHashTxnSize = 32768;
```

Comment:
What is the meaning/reasoning behind this constant?

Reply:
if the tx rlp is above it we calculate hash eagerly (probably in order not to keep that memory longer on rented array), if below we try to do it in the background.

Reply:
My question was more targeted to the reasoning behind that particular constant. Was it decided based on some benchmark/test, or does it have a particular meaning?